kernel/kernel-bioapi-provider/src/main/java/io/mosip/kernel/bioapi/impl/BioApiImpl.java

```diff
@@ -58,8 +58,10 @@ public QualityScore checkQuality(BIR sample, KeyValuePair[] flags) {
 			if (Objects.nonNull(recordedValue) && Objects.nonNull(recordedValue.getBdb())
 					&& recordedValue.getBdb().length != 0 && Arrays.equals(recordedValue.getBdb(), sample.getBdb())) {
 				matchingScore[count].setInternalScore(90);
+				matchingScore[count].setScaleScore(90);

```

Comment:
Hi Nagarjuna, Please create a common local variable for the expressions used in the setters and use the same in both setters